Makefile

```diff
@@ -14,6 +14,17 @@ BASE=$(GOPATH)/src/$(REPO_PATH)
 GOFILES = $(shell find . -name *.go | grep -vE "(\/vendor\/)|(_test.go)")
 PKGS     = $(or $(PKG),$(shell cd $(BASE) && env GOPATH=$(GOPATH) $(GO) list ./... | grep -v "^$(PACKAGE)/vendor/"))
 TESTPKGS = $(shell env GOPATH=$(GOPATH) $(GO) list -f '{{ if or .TestGoFiles .XTestGoFiles }}{{ .ImportPath }}{{ end }}' $(PKGS))
+SRCROOT = $(abspath $(dir $(lastword $(MAKEFILE_LIST)))/)
+BUILD_ROOT = $(SRCROOT)/build
+
+UPSTREAM_VERSION=$(shell git describe --tags HEAD)
```

Comment:
for upstream/tag version can we use something like this ?
basically try to strip off  `4-g7a91015` in v2.6.2-4-g7a91015-pmk-2353472 ? 
to keep only v2.6.2-pmk-2353472 so that it can be trackable using teamcity build_id ?

Reply:
Okay, will make the changes